packages/datadog-plugin-grpc/src/server.js

```diff
@@ -0,0 +1,136 @@
+'use strict'
+
+const Tags = require('../../../ext/tags')
+const TEXT_MAP = require('../../../ext/formats').TEXT_MAP
+const kinds = require('./kinds')
+const { addMethodTags, addMetadataTags, getFilter } = require('./util')
+
+function handleError (span, err) {
```

Comment:
Are the `err`'s a typeof `Error`? If so, we can just let the formatter handle the error tags.